Makefile

```diff
@@ -102,6 +102,7 @@ dkms_modules_install: dkms_check
 	dkms add .
 	dkms build $(DKMS_MODULES)
 	dkms install $(DKMS_MODULES)
+	@! rmmod hid_uclogic
```

Comment:
I'm hesitant about loading or unloading modules as part of (un)installation, since we don't really know which version of the module is loaded at that moment. At the very least this should be a part of an uninstall target, but I'd prefer, if you dropped this patch.